lib/gqls/resolvers/leave_request.dart

```diff
@@ -0,0 +1,67 @@
+const String leaveRequestsListGQL = """
+  query (\$input: PagyInput, \$query: LeaveDayRequestsQuery) {
+    LeaveDayRequests(input: \$input, query: \$query) {
+      collection {
+        id
+        userId
+        approverId
+        from
+        to
+        timeOff
+        requestType
+        requestState
+        reason
+        createdAt
+        updatedAt
+        lockVersion
+        User {
+          id
+          email
+          fullName
+          name
+          about
+          avatarUrl
+          createdAt
+          companyLevelId
+          state
+          address
+          birthday
+          gender
+          phone
+          timingActiveAt
+          timingDeactiveAt
+          slackId
```

Comment:
có cần thiết phải lấy toàn bộ như này không
dùng gql thì nên hạn chế lấy data không cần thiết